tabpy-tools/tabpy_tools/rest.py

```diff
@@ -214,7 +214,7 @@ def __delete__(self, instance):
         delattr(instance, self.name)
 
 
-class _RESTMetaclass(type(_MutableMapping)):
+class _RESTMetaclass(abc.ABCMeta):
```

Comment:
Not sure the intention of this class but most metaclasses would define __new__ instead of __init__. Actually not sure the intentions of the latter but makes it seem like more of a base class than a meta